rollup.config.cjs

```diff
@@ -0,0 +1,9 @@
+const { nodeResolve } = require("@rollup/plugin-node-resolve");
+
+module.exports = {
+  input: "src/cli.js",
```

Comment:
I haven't tried to run the build, but this config can't be right because this file doesn't exist.

Can you run the build and verify it works? If it does work, I think it means we're don't need this `input` configuration at all and should remove it so it doesn't confuse us later.